system-design/specification/applications/application-registry.md

```diff
@@ -0,0 +1,259 @@
+# Application Registry
+
+The interactions of an `Application Registry` are described [here](../../concepts/applications/application-registry.md) from a conceptual view. This section formally specifies the API of the `Application Registry` and the exchange of an [Application Package](../../concepts/applications/application-package.md), defined through an [Application Description](../../specification/applications/application-description.md) file, from an Application Developer to the [Workload Fleet Manager](../../personas-and-definitions/technical-lexicon.md#workload-fleet-manager) (WFM). The `Application Registry` is designed as an `OCI Registry`, i.e., it offers an API compliant with the [OCI Registry API (v1.1.0)](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md) (called here the "OCI_spec") for digital artifact distribution. This way, the Application Registry hosts the parts of an [Application Package](../../concepts/applications/application-package.md) in the form of `blobs` and uses `image manifests` according to [OCI Image specification](https://github.com/opencontainers/image-spec/blob/v1.0.1/manifest.md) to list a set of `layers`, each pointing at a blob.
```

Comment:
The OCI specification has different sub-modules. You are introducing features here available in the OCI Image spec v1.1 and you are referencing that right. For the distribution though the features of the OCI Distribution (API) spec v1.0 suffice. Therefore I would simply leave the version for the OCI Distribution spec "underspecified" (so no version given).

Why? Because although all the registries that I know (GitHub, GitLab, Harbor, ZOT, Docker Registry,...) provide all the needed functionality, only one of them (ZOT) conforms 100% with the OCI Distribution spec v1.1. Someone validating compatibility "on paper" might then think that only one registry can be used, what is not the case.